docs/archives.md

```diff
@@ -105,7 +105,18 @@ You can pass a fourth parameter called `options` to `withArchive`. This is an ob
 
 ## Pagination
 
-Pagination support for archives is included out of the box. To load the next page in an archive, simply call the `onLoadMore` prop passed in by `withArchive`.
+Pagination support for archives is included out of the box. You can use either infinite scroll style ("more") or manual pagination.
+
+With more pagination, your component will receive all posts Repress has loaded. When you load more posts, it will append these to the list of posts, and your component will receive this via the `posts` prop.
```

Comment:
It seems this should be `"more"` pagination, or More Pagination? It reads quite confusingly when it's read as "With extra pagination"

Reply:
I'll rename to `more-style pagination`.